products/data.js

```diff
@@ -0,0 +1,54 @@
+export const beans = [
+    {
+        id: '0',
+        name: 'Scarlet runner',
+        image: 'scarletr.jfif',
+        type: 'a large pole variety',
+        price:  3,
+        germination: '7 to 14'
+
+
```

Comment:
in the interest of readability/flow, try to keep excess white space to a minimum, using single blank lines to separate stanzas in your code poetry